### PR TITLE
pr-tool: give hint about commit headline length limits

### PR DESCRIPTION
**Pull request by joergsteffens**
### Thank you for contributing to the Bareos Project!

I already wondered several times, about how long headline could be. Changed the code to give a hint. Decided not to mention the 50, to keep the remark as short as possible.

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)
- [Boy Scout Rule](https://docs.bareos.org/DeveloperGuide/generaldevel.html#boy-scout-rule)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
~~Required backport PRs have been created~~

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

### CODE REVIEW

File: devtools/pip-tools/pr_tool/main.py [hunk 1]
```diff
@@ -184,7 +184,7 @@ def _check_headline(cls, text):
         issues = []
         # we encourage to use no more than 50 chars, but still accept up to 60
         if len(text) > 60:
-            issues.append("headline too long")
+            issues.append("headline too long ({} > 60)".format(len(text)))
```
Comment (sebsura) [inline devtools/pip-tools/pr_tool/main.py:187]: The 60 should get named, so we cannot accidentally only change it in one place
```
max_headline_length = 60
if len(text) > max_headline_length
    issues.append("headline too long ({} > {})".format(len(text), max_headline_length))
	
```

Comment (joergsteffens) [reply devtools/pip-tools/pr_tool/main.py]: done.

**Review by sebsura — APPROVED** [review]
Nice change!

**Comment by sebsura** [comment]
Could you fix the type in the first commit headline ? `lenght` -> `length`

**Comment by joergsteffens** [comment]
Fixed type and squashed commits.

**Pull request merged**